app/views/bulk_discounts/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>New Bulk Discount</h1>
+
+<section id="new_discount">
+    <%= form_with url: "/merchants/#{@merchant.id}/bulk_discounts", method: :post, data: {turbo: false} do |f| %>
+     <fieldset>
```

Comment:
This was new to me, but Terry showed me this and it creates a border to fit around the form, which makes it possible to test `expect(find("form")).to have_content("Add Percentage Discount")`